pkg/caller/frame.go

```diff
@@ -13,6 +14,8 @@ type Frame struct {
 	Line     int
 }
 
+var kanisterPathRe = regexp.MustCompile(`/kanister(@v[^/]+)?/`)
```

Comment:
[nit] This is pretty readable, but with regex, it's often useful to provide an example of what you'll match against.